README.md

```diff
@@ -415,22 +417,19 @@ Build the cppzmq bindings.
 We assume you are compiling with a non-root user and you have `doas` enabled.
 
 ```
-# Create a library link so cmake is able to find it
-doas ln -s /usr/local/lib/libzmq.so.4.1 /usr/local/lib/libzmq.so
-
 # Create cppzmq building directory
 mkdir ~/cppzmq
 cd ~/cppzmq
 
 # Fetch cppzmq source
-ftp -o cppzmq-4.2.2.tar.gz https://github.com/zeromq/cppzmq/archive/v4.2.2.tar.gz
+ftp -o cppzmq-4.2.3.tar.gz https://github.com/zeromq/cppzmq/archive/v4.2.3.tar.gz
 
-# MUST output: (SHA256) cppzmq-4.2.2.tar.gz: OK
-echo "3ef50070ac5877c06c6bb25091028465020e181bbfd08f110294ed6bc419737d cppzmq-4.2.2.tar.gz" | sha256 -c
-tar xfz cppzmq-4.2.2.tar.gz
+# MUST output: (SHA256) cppzmq-4.2.3.tar.gz: OK
+echo "3e6b57bf49115f4ae893b1ff7848ead7267013087dc7be1ab27636a97144d373 cppzmq-4.2.3.tar.gz" | sha256 -c
```

Comment:
Shouldn't these have somehting like || exit 1 (can be in another PR as it's existing) ?

Reply:
Usually, yes, if this would be a shell script.

But since this is a step by step copy/paste guide, adding a `|| exit 1` would just log out the users session.

Reply:
Ah, fair point :)